tests/unit/example.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { shallowMount } from '@vue/test-utils'
```

Comment:
add test for signout and signin views too? so that we can set the standard to mandate test for each component.

Reply:
I will add more test case